### PR TITLE
Release 6.0.0rc1

### PR DESCRIPTION
**Pull request by nicoddemus**
As discussed in https://github.com/pytest-dev/pytest/issues/7447.

### CODE REVIEW

File: doc/en/announce/release-6.0.0rc1.rst [hunk 1]
```diff
@@ -19,7 +19,6 @@ Thanks to all who contributed to this release, among them:
 * Bruno Oliveira
 * Claire Cecil
 * Curt J. Sampson
-* Daniel
 * Daniel Hahler
```
Comment (blueyed) [inline doc/en/announce/release-6.0.0rc1.rst:22]: That's not a duplicate (6cbbd2d).

Comment (nicoddemus) [reply doc/en/announce/release-6.0.0rc1.rst:22]: I will fix it, thanks.

File: doc/en/changelog.rst [hunk 2]
```diff
@@ -28,6 +28,420 @@ with advance notice in the **Deprecations** section of releases.
...
+    been removed; use plain ``compile()`` instead.
+  - The function ``_pytest._code.source.getsource()`` has been removed; use
+    ``Source()`` directly instead.
+
+
+
+Deprecations
+------------
+
+- `#6981 <https://github.com/pytest-dev/pytest/issues/6981>`_: Deprecate the ``pytest.collect`` module as it's just aliases into ``pytest``.
```
Comment (blueyed) [inline doc/en/changelog.rst:137]: There is a known regression with this already: https://github.com/pytest-dev/pytest/pull/6981#issuecomment-606496652
(IIRC it does not happen with https://github.com/pytest-dev/pytest/pull/6803)

**Comment by hugovk** [comment]
Is #7362 ("Change pytest deprecation warnings into errors for 6.0 release") needed for this?

**Comment by nicoddemus** [comment]
> Is #7362 ("Change pytest deprecation warnings into errors for 6.0 release") needed for this?

I think so yes, this way release testers can try removing the warnings and reporting any problems. 👍

File: doc/en/getting-started.rst [hunk 3]
```diff
@@ -28,7 +28,7 @@ Install ``pytest``
 .. code-block:: bash
 
     $ pytest --version
-    This is pytest version 5.x.y, imported from $PYTHON_PREFIX/lib/python3.8/site-packages/pytest/__init__.py
+    pytest 5.4.1.dev624+g4f318f86b.d20200707
```
Comment (asottile) [inline doc/en/getting-started.rst:31]: we probably need to do something in regen for this

Comment (nicoddemus) [reply doc/en/getting-started.rst:31]: Oh good catch, missed that one. I will take a look later.

Comment (nicoddemus) [reply doc/en/getting-started.rst]: Fixed, thanks!

Now it shows the actual version. 👍

**Comment by nicoddemus** [comment]
Unfortunately `TestParseIni::test_missing_required_plugins` depends on `1.32.0` being the last pytest-xdist release, so `1.33.0` which I *just* published specifically to release `6.0.0rc1` broke the tests.

I will make the release manually for now, given that this is really a problem with the tests and not with the code, and will fix the tests shortly in a separate PR.

**Comment by asottile** [comment]
@nicoddemus looks like the github release notes got missed: https://github.com/pytest-dev/pytest/releases/tag/6.0.0rc1

I assume this is related to the manual release?

**Comment by nicoddemus** [comment]
Ahh right, indeed it is, thanks for the heads up. 

Given that this is a prerelease I'm willing to leave it at that for now, unless someone really complains about it. 😁

**Comment by asottile** [comment]
alrighty sounds good to me!

**Pull request merged**